### ISSUE TITLE
[spec] Allow multiple lookups per glyph class?

### ISSUE THREAD

**Issue by brawer** [adobe-type-tools/afdko#108]
[Section 5.f.i](http://www.adobe.com/devnet/opentype/afdko/topic_feature_file_syntax.html#5.f.i) of the OpenType Feature File Specification says currently:
> For each glyph or glyph class in the input sequence, the contextual rule may specify one lookup (§4.e) to be applied at that position.

However, in the GSUB table, [SubstLookupRecord](https://www.microsoft.com/typography/otspec/gsub.htm#CS) can call multiple lookups on the same SequenceIndex. Could the feature file syntax be changed to allow multiple lookups for the same glyph class? Perhaps like this:
```
substitute [a e i o u]' lookup L1, L2, L3 f' i' n' lookup L2, L1;
```

Likewise for positioning. This is certainly a corner case, but it might be useful for decompilers if all valid OpenType tables can expressed as a feature file.

**Comment by behdad** [comment]
The format itself allows even out-of-order application of lookups.

**Comment by behdad** [comment]
Also, there are cases where, the invoked lookup will ligate things, in which case the indices of subsequent actions reflects positions of the new glyphs.

**Comment by readroberts** [comment]
Yes, this is a good idea. We should change the spec and implementations to support this.
